src/aznfswatchdog

```diff
@@ -131,13 +131,26 @@ reconcile_conntrack()
 
 vecho "Starting aznfswatchdog..."
 
+# Detect and log distro and bash version
+if [ -f /etc/centos-release ]; then
+    linux_distro=$(cat /etc/centos-release)
```

Comment:
If /etc/centos-release is present then it is centos only. Theerefore no need to do cat and just assign the value.

Reply:
To get complete version number. 
example: 
]# cat /etc/centos-release
CentOS Linux release 8.5.2111